test/testsuite/linalg.jl

```diff
@@ -221,12 +221,13 @@
         @test compare(lmul!, AT, Ref(rand(T)), rand(T, b))
     end
 
-    @testset "axp{b}y" for T in eltypes
-        alpha, beta = 0.5, 2.0
-        x = T.([2,4,6])
-        y = T.([3,4,5])
-        @test axpby!(alpha,x,beta,y) ≈ T.([7,10,13])
-        @test axpy!(alpha,x,y) ≈ T.([8,12,16])
+    @testset "axp{b}y" for T in (Float32, ComplexF32)
```

Comment:
That's not correct, you still need to respect `eltypes`. Furthermore, oneAPI.jl already restricts that to supported ones, https://github.com/JuliaGPU/oneAPI.jl/blob/68e0d9d12b61dee66cb5e0185a6cc7c329a0fa35/test/setup.jl#L25. Isn't this test supposed to work on integers?

Reply:
Ok, I tested locally with
```
T = oneArray
TestSuite.test_linalg(T)
```
and all tests passed.
`axpy!` and `axpby!` are BLAS1 routines and are supposed to work with real and complex numbers.
We also test them with integers but these routines are generally used with `BlasFloat` types.

Reply:
The test can restrict which element types to use, but it needs to respect the `eltypes` argument (which reflects hardware support, and is set by the package that uses the test suite).